examples/deployment-with-init/deployment.tf

```diff
@@ -0,0 +1,51 @@
+
+# Create an Elastic Cloud deployment
+resource "ec_deployment" "example_minimal" {
+  # Optional name.
+  name = "my_example_deployment"
+
+  # Mandatory fields
+  region                 = "us-east-1"
+  version                = "7.9.2"
+  deployment_template_id = "aws-io-optimized-v2"
+  traffic_filter = [ec_deployment_traffic_filter.allow_all.id]
+  elasticsearch {
+    topology {
+      instance_configuration_id = "aws.data.highio.i3"
```

Comment:
The `instance_configuration_id` fields are no longer mandatory, they are now self discoverable 🎉 If you want to take them out you can :)

Reply:
Yeah I think omitting this is positive since doesn't add anything useful